eos/b-decays/b-to-vec-nu-nu.cc

```diff
@@ -0,0 +1,275 @@
+        // process: string that can be used to obtain the form factor
+        // B: name of the B meson
+        // V: name of the daughter meson
+        // c_I: isospin factor by which the amplitudes are multiplied
+        static const std::map<std::tuple<QuarkFlavor, QuarkFlavor, std::string>, std::tuple<std::string, std::string, std::string, double>> process_map;
+
+        inline std::string _process() const
+        {
+            const auto D = opt_D.value();
+            const auto q = opt_q.value();
+            const std::string I = opt_I.value();
+            const auto p = process_map.find(std::make_tuple(D, q, I));
+
+            if (p == process_map.end())
+                throw InternalError("Unsupported combination of D=" + opt_D.str() + ", q=" + opt_q.str() + ", I=" + I);
+
+            return std::get<0>(p->second);
+        }
+
+        inline std::string _B() const
+        {
+            const auto D = opt_D.value();
+            const auto q = opt_q.value();
+            const std::string I = opt_I.value();
+            const auto p = process_map.find(std::make_tuple(D, q, I));
+
+            if (p == process_map.end())
+                throw InternalError("Unsupported combination of D=" + opt_D.str() + ", q=" + opt_q.str() + ", I=" + I);
+
+            return std::get<1>(p->second);
+        }
+
+        inline std::string _V() const
+        {
+            const auto D = opt_D.value();
+            const auto q = opt_q.value();
+            const std::string I = opt_I.value();
+            const auto p = process_map.find(std::make_tuple(D, q, I));
+
+            if (p == process_map.end())
+                throw InternalError("Unsupported combination of D=" + opt_D.str() + ", q=" + opt_q.str() + ", I=" + I);
+
+            return std::get<2>(p->second);
+        }
+
+        inline double _isospin_factor() const
+        {
+            const auto D = opt_D.value();
+            const auto q = opt_q.value();
+            const std::string I = opt_I.value();
+            const auto p = process_map.find(std::make_tuple(D, q, I));
+
+            if (p == process_map.end())
+                throw InternalError("Unsupported combination of D=" + opt_D.str() + ", q=" + opt_q.str() + ", I=" + I);
+
+            return std::get<3>(p->second);
+        }
+
+
+        Implementation(const Parameters & p, const Options & o, ParameterUser & u) :
+            model(Model::make(o.get("model", "SM"), p, o)),
+            parameters(p),
+            opt_D(o, options, "D"),
+            opt_q(o, options, "q"),
+            opt_I(o, options, "I"),
+            m_B(p["mass::" + _B()], u),
+            tau_B(p["life_time::" + _B()], u),
+            m_V(p["mass::" + _V()], u),
+            g_fermi(p["WET::G_Fermi"], u),
+            alpha_e(p["QED::alpha_e(m_b)"], u),
+            hbar(p["QM::hbar"], u),
+            isospin_factor(_isospin_factor()),
+            mu(p[opt_D.str() + "b" + "nunu::mu"], u),
```

Comment:
Since the WilsonCoefficient object doesn't depend on `mu`, why do you have this parameter here?

Reply:
The matrix elements do depend on it, in general. It's not applicable here, but hadronic matrix element of the scalar and/or tensor operators are explicitly mu dependent.

Reply:
understood!